spk/readarr/src/service-setup.sh

```diff
@@ -0,0 +1,34 @@
+
+# Readarr service setup
+
+READARR="${SYNOPKG_PKGDEST}/share/Readarr/bin/Readarr"
+
+# Readarr uses custom Config and PID directories
```

Comment:
I suppose the comment should tell "config and home directories" as there is not PID-File definition below